portfolio/src/main/webapp/index.html

```diff
@@ -1,18 +1,75 @@
 <!DOCTYPE html>
 <html>
+    <style>
+    
+    body {
+    background-image: url('imgDir/black-close-up-dark-dew-396547.jpg');
+    background-repeat: no-repeat;
+    background-attachment: fixed; 
+    background-size: cover;
+    color: white;
+}
```

Comment:
make sure to align this closing bracket inwards more so it looks like this:
body {
  backgroud-image....
  ....
}

Also, indent the properties 2 spaces in too!

Reply:
Fix for the following below as needed